README.md

```diff
@@ -447,6 +451,10 @@ The `start` and `end` properties both refer to the current parse position. The
 `offset` property contains an offset as a zero-based index and `line` and
 `column` properties contain a line and a column as one-based indices.
 
+Line and column are somewhat expensive to compute, so if you just need the
```

Comment:
should we refactor the readme so that this information doesn't need to be in two places?